packages/attestation-service/README.md

```diff
@@ -0,0 +1,33 @@
+# Attestation Service
+
+This is the attestation service that validators run on the Celo network to provide attestations of phone numbers to accounts.
+
+### Configuration
+
+You can use the following environment variables to configure the attestation service:
+
+- `DATABASE_URL` - The URL under which your database is accessible, currently supported are `postgres://`, `mysql://` and `sqlite://`
+- `CELO_PROVIDER` - The URL under which a celo blockchain node is reachable, i.e. something like `https://integration-forno.celo-testnet.org`
+- `ACCOUNT_ADDRESS` - The address of the account on the `Accounts` smart contract
```

Comment:
Can we get this from contract kit?

Reply:
I can see how the phrasing is confusing. It's not the address of the `Accounts` smart contract, but the account address of the validator account on the `Accounts` smart contract. Not sure what a better name/description would be